app/src/main/java/me/simplq/MainActivity.java

```diff
@@ -1,14 +1,111 @@
 package me.simplq;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.app.ActivityCompat;
 
+import android.Manifest;
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.content.pm.PackageManager;
 import android.os.Bundle;
+import android.telephony.SmsManager;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.google.android.material.snackbar.Snackbar;
 
 public class MainActivity extends AppCompatActivity {
+    Button btnSendSMS;
+    EditText txtPhoneNo;
+    EditText txtMessage;
+    private View mLayout;
+    private static final int PERMISSION_REQUEST_SMS = 0;
+
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        mLayout = findViewById(android.R.id.content);;
```

Comment:
I suppose we're not using any formatter 😅

Reply:
Have to set it up...